### PR TITLE
feat(Flutter Plugin): add detailed delivery receipt data to the plugin

### PR DESCRIPTION
**Pull request by juansoos**
Relates to https://github.com/la-haus/app-adviser-flutter/issues/416

This implementation adds new models `DetailedDeliveryReceiptData` and `DetailedDeliveryReceipt` in addition to generating new code and logic able to do the mapping between the flutter and channels (Android-iOS)

### CODE REVIEW

**Review by danielgomezrico — APPROVED** [review]
Nice!!!!!

File: lib/src/message/message.dart [hunk 1]
```diff
@@ -151,4 +152,18 @@ class Message {
   }
 
 //TODO: implement getDetailedDeliveryReceiptList
```
Comment (cdmunoz) [inline lib/src/message/message.dart]: This TODO could be removed already, WDYT?

**Review by cdmunoz — APPROVED** [review]
Osom 💯

File: lib/src/message/message.dart (hunk 1 above)
Comment (juansoos) [reply lib/src/message/message.dart]: Removed ✅

**Pull request merged**